### PR TITLE
OpenMPI: add support for pbs/tm at configure.

### PR DESCRIPTION
**Pull request by bebosudo**
Hi!
I'd like to add support to PBS/TM to OpenMPI, as it's already done with the mellanox mxm drivers.

### CODE REVIEW

File: var/spack/repos/builtin/packages/openmpi/package.py [hunk 1]
```diff
@@ -332,7 +344,18 @@ def with_or_without_mxm(self, activated):
...
         line = '--with-{0}'.format(opt)
         path = _mxm_dir()
-        if (path is not None):
+        if path is not None:
+            line += '={0}'.format(path)
+        return line
+
+    def with_or_without_tm(self, activated):
+        opt = 'tm'
+        # If the option has not been activated return --without-mxm
```
Comment (jrood-nrel) [inline var/spack/repos/builtin/packages/openmpi/package.py]: This comment didn't get updated.

Comment (bebosudo) [reply var/spack/repos/builtin/packages/openmpi/package.py]: Fixed!

**Comment by bebosudo** [comment]
Hi @jrood-nrel, any news?

**Comment by jrood-nrel** [comment]
Sorry @bebosudo , if you're looking for this to be merged, I don't have merge access for Spack.

**Comment by bebosudo** [comment]
Hi @tgamblin, can I ask you to review this patch?
I often find myself patching it on every installation and pointing people at it to have a usable openmpi+pbs system.

**Comment by navdeeprana** [comment]
@adamjstewart Can you review this patch? I didn't find any other way of making openmpi work with pbs.

**Comment by scheibelp** [comment]
Thanks!

**Pull request merged**